bin/live/pycbc_live_combine_single_fits

```diff
@@ -149,22 +157,27 @@ for ifo in args.ifos:
         c = c[valid_alpha]
         invalphan = c / a
         mean_alpha = c.mean() / invalphan.mean()
-        q05_alpha = np.quantile(a, 0.05)
-        q05_alphas_out[ifo][counter] = q05_alpha
+        cons_alpha = np.quantile(a, qlower)
+        cons_alphas_out[ifo][counter] = cons_alpha
         alphas_out[ifo][counter] = mean_alpha
-        q95_count = np.quantile(c, 0.95)
-        q95_counts_out[ifo][counter] = q95_count * len(c)
+        cons_count = np.quantile(c, qupper)
+        cons_counts_out[ifo][counter] = cons_count * len(c)
```

Comment:
What about using `np.percentile()` here instead of `np.quantile()`, and avoiding having to carry around `qlower` and `qupper`?

Reply:
That makes sense; it looks a little more clunky given the inline int conversion, but it saves a variable or two